pyproject.toml

```diff
@@ -143,6 +143,9 @@ select = [
   "FA100",  # Missing from __future__ import annotations
   "W605",   # Invalid escape sequences
 ]
+ignore = [
+  "A003"    # Class attribute shadowing a Python builtin
+]
```

Comment:
Ignoring flake8 A003 and removing some `noqa: A003` comments. This prevents us from adding a `type` field on Entity classes. Using an alias prevents us from using discriminators (not supported by Pydantic).

Reply:
I don't like it, it's a known source of errors. It's not also limited to `type` but also other builtins. 

I think `noqa: A003` on single line in model (well, every model) is safer.

Reply:
That's fair. I tried to avoid having `noqa` on each entity type. Reverted.